tests/templates/__init__.py

```diff
@@ -35,62 +36,103 @@ class Response(NamedTuple):
 
 class ImplRequest:
     """
-    Implementation of a org.freedesktop.impl.portal.Request object. Typically
-    this object needs to be merely exported:
+    Implementation of a ``org.freedesktop.impl.portal.Request`` object exposed
```

Comment:
```suggestion
    Implementation of an ``org.freedesktop.impl.portal.Request`` object exposed
```

might as well change this while here :)